Makefile

```diff
@@ -52,23 +53,8 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_BUILD) \
 --enable-decoder=h264 \
 --enable-decoder=hevc \
 --enable-decoder=mjpeg \
---enable-demuxer=concat \
---enable-demuxer=mov \
---enable-demuxer=mp4 \
---enable-demuxer=segment \
---enable-encoder=libx264 \
 --enable-encoder=mjpeg \
---enable-encoder=mpeg4 \
---enable-gpl \
---enable-libx264 \
---enable-muxer=mp4 \
---enable-muxer=segment \
---enable-network \
---enable-parser=h264 \
---enable-parser=hevc \
---enable-protocol=concat \
---enable-protocol=crypto \
---enable-protocol=file \
```

Comment:
pretty sure we still need these on `not` windows in order for video-store to work

Reply:
Added everything back, there should be no change to base functionality FFmpeg configure flags.